<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labels</class>
 <widget class="QDockWidget" name="labels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>175</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>150</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFeatureMask</set>
  </property>
  <property name="windowTitle">
   <string>Labels</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="1" column="2">
     <widget class="QPushButton" name="pbRemove">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="cbEnableEditing">
      <property name="text">
       <string>Enable Editing</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pbStoreImage">
      <property name="text">
       <string>Store Image</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pbLoad">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pbStore">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Store</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="7">
     <widget class="QTreeView" name="lvLabels">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pbAdd">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lvLabels</tabstop>
  <tabstop>pbAdd</tabstop>
  <tabstop>pbRemove</tabstop>
  <tabstop>cbEnableEditing</tabstop>
  <tabstop>pbStoreImage</tabstop>
  <tabstop>pbStore</tabstop>
  <tabstop>pbLoad</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
